src/lxilua.c

```diff
@@ -169,6 +169,14 @@ static int disconnect(lua_State *L)
     int status = 0;
     int device = lua_tointeger(L, 1);
 
+    #ifdef __CYGWIN__
```

Comment:
You can remove the #ifdef __CYGWIN__ stuff. This fix is valid for all platforms.